mautic/mautic
# Forms contact tab

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/265
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

Bugfix/feature donate by www.newcom4u.ch

[//]: # ( Required: )
#### Description:
PR added forms as tab to contacts detail page. 
Also PR add new option to forms - "Show results in contacts tab"
This feature allow users add relevant forms contact result to details (for example - orders,  reservation etc)

![image](https://user-images.githubusercontent.com/462477/35951093-2ba4fb28-0c7a-11e8-966e-098381a4ab0f.png)

![image](https://user-images.githubusercontent.com/462477/35966638-b9b08b32-0cbe-11e8-8f98-9f18de3b7d05.png)

#### Steps to test this PR:
1.  Apply PR and run DB migration `php app/console doctrine:migrations:migrate` or `php app/console doctrine:schema:update --force` 
2.  Create form and enable option "Show results in contacts tab"
3. Send form 
4. Go to contacts detail page and see if new tab Forms is showed with relevant data
5. Test case If contact don't have any forms fill
6. Test case If Mautic user don't have permission don't see forms
